package.json

```diff
@@ -22,59 +22,43 @@
     "bi-app-sass": "^1.1.0",
     "bourbon": "^4.2.6",
     "breakpoint-sass": "^2.6.1",
+    "font-awesome": "^4.6.3",
     "jquery": "^1.11.1",
     "requirejs": "^2.1.22",
     "requirejs-plugins": "^1.0.2",
-    "sizzle": "2.3.0",
-    "susy": "^2.2.9"
+    "susy": "^2.2.9",
+    "webpack-dev-server": "^1.15.2"
   },
   "peerDependencies": {
     "font-awesome": "^4.6.3"
   },
   "devDependencies": {
     "backbone": "*",
-    "browser-sync": "*",
     "css-loader": "~0.23.1",
     "del": "*",
     "edx-ui-toolkit": "~1.4.1",
     "eslint": "~2.13.1",
-    "eslint-config-edx": "~1.2.0",
-    "extract-text-webpack-plugin": "~1.0.1",
-    "file-loader": "0.8.5",
-    "font-awesome-loader": "0.0.1",
+    "eslint-config-edx": "~1.2.1",
+    "extract-text-webpack-plugin": "^1.0.1",
+    "file-loader": "^0.9.0",
     "gulp": "*",
-    "gulp-changed": "*",
-    "gulp-concat": "*",
-    "gulp-eslint": "^2.0.0",
-    "gulp-filesize": "*",
-    "gulp-filter": "*",
     "gulp-gh-pages": "~0.5.4",
-    "gulp-ignore": "2.0.1",
-    "gulp-minify-css": "*",
     "gulp-notify": "*",
-    "gulp-rename": "*",
     "gulp-shell": "^0.5.2",
-    "gulp-uglify": "*",
     "gulp-util": "*",
-    "lodash": "*",
-    "merge-stream": "*",
-    "minimist": "*",
-    "node-sass": "~3.7.0",
+    "node-sass": "^3.10.0",
     "pretty-hrtime": "^1.0.2",
     "require-dir": "*",
-    "run-sequence": "*",
-    "sass-loader": "~3.2.0",
-    "style-loader": "~0.13.1",
-    "text-loader": "*",
-    "underscore": "~1.8.3",
-    "url-loader": "^0.5.6",
+    "sass-loader": "~4.0.2",
+    "style-loader": "^0.13.1",
+    "text-loader": "0.0.1",
+    "url-loader": "^0.5.7",
     "webpack": "~1.12.14",
     "webpack-stream": "~3.2.0"
   },
   "scripts": {
-    "production": "gulp build-production",
     "gulp": "gulp",
-    "development": "gulp build-development",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "start": "rm -rf pldoc/public && jekyll serve --incremental & webpack-dev-server && fg"
```

Comment:
`"scripts": {}` is a pretty common thing for a package to configure build/development tasks if it's not using another tool like gulp, so I guess the question is does having an `npm start` command that does the same thing as `$(npm bin)/gulp` add any value?

(I added this because I'm not a huge Gulp fan and I kind of just wanted to prove we didn't need it to do the development workflow... so having proved my point, I'll remove this and just make it an alias on my machine if nobody but me is going to use it 😛)